e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -299,6 +299,14 @@ describe('Plugin Marketplace', () => {
 
         it('change tab on "install plugins" click', () => {
             cy.get('#marketplaceTabs').should('exist').within(() => {
+                // # uninstall all user`s plugins
+                cy.getAllPlugins().then((response) => {
+                    const active = response.body.active;
+                    const inactive = response.body.inactive;
+                    inactive.forEach((plugin) => cy.uninstallPluginById(plugin.id));
+                    active.forEach((plugin) => cy.uninstallPluginById(plugin.id));
+                });
```

Comment:
Could we move this to the `after` call for the set of tests that installed plugins so it can make sure that it cleanups and does not affect any other tests? 

I had added a remove for specific plugins [here](https://github.com/mattermost/mattermost-webapp/blob/e568db53a29447e1cb01875a93ff1f9800dd7b57/e2e/cypress/integration/plugins/marketplace/marketplace_spec.js#L121) and [here](https://github.com/mattermost/mattermost-webapp/blob/e568db53a29447e1cb01875a93ff1f9800dd7b57/e2e/cypress/integration/plugins/marketplace/marketplace_spec.js#L210-L213) but I believe your approach is more robust and just uninstalls everything. Could you update that to use your implementation?